pubspec.yaml

```diff
@@ -64,18 +64,7 @@ flutter:
   uses-material-design: true
 
   assets:
-    - assets/registerpagegradient.png
```

Comment:
Just want to make sure we don't break anything. Is everything still working as expected with this change?